eyeshade/lib/queries.js

```diff
@@ -87,3 +88,16 @@ WHERE
     geo_referral_groups.active_at <= $1
 AND countries.group_id = geo_referral_groups.id;`
 }
+
+function getActiveCountryGroups () {
```

Comment:
weird formatting where some of the postgres keywords are uppercase, some lowercase.

Reply:
fixed in: eec22ca